qrexec/client.py

```diff
@@ -102,11 +101,6 @@ def make_command(dest, rpcname, arg):
     if arg is not None:
         rpcname = f"{rpcname}+{arg}"
 
-    if VERSION == "dom0" and dest == "dom0":
-        # Invoke qubes-rpc-multiplexer directly. This will work for non-socket
-        # services only.
-        return [RPC_MULTIPLEXER, rpcname, "dom0"]
-
```

Comment:
Well, this needs a replacement, otherwise calling dom0->dom0 services fail. For example policy editor crashes with:
```
subprocess.CalledProcessError: Command '['/usr/bin/qrexec-client', '-d', 'dom0', 'DEFAULT:QUBESRPC policy.List dom0']' returned non-zero exit status 1.
```